content/influxdb/clustered/admin/databases/_index.md

```diff
@@ -44,17 +44,70 @@ never be removed by the retention enforcement service.
 In {{< product-name >}}, table (measurement) and column limits can be
 custom configured when [creating](#create-a-database) or
 [updating a database](#update-a-database).
-Each measurement is represented by a table.
-Time, fields, and tags are each represented by a column.
 
-**Default maximum number of tables**: 500  
+### Table limit
+
+**Default maximum number of tables**: 500
```

Comment:
We'll also need to update https://docs.influxdata.com/influxdb/clustered/write-data/best-practices/schema-design/#measurements-can-contain-up-to-200-columns